keycloak/README.md

```diff
@@ -0,0 +1,6 @@
+## Purpose
+
+This common action covers the one of the limitations of Terraform, such that there is no way to control of existing resources, and ensures the target Keycloak instances to be provisioned as:
+
+1. Disable `review profile config` execution of the default authentication flow `first broker login`.
```

Comment:
I don't really understand what these enumerated points mean.  Are these the the provisioning steps mentioned in the first sentence, or are these the instances being provisioned?

Reply:
hm. I updated the description. hope it is a little bit clearer.